.devcontainer/Dockerfile

```diff
@@ -30,27 +30,27 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update -q && \
 RUN source "/opt/ros/${ROS_DISTRO}/setup.bash"
 
 # Install bosdyn_msgs package
-RUN curl -sL https://github.com/bdaiinstitute/bosdyn_msgs/releases/download/4.0.2/ros-humble-bosdyn_msgs_4.0.2-jammy_amd64.run --output /tmp/ros-humble-bosdyn_msgs_4.0.2-jammy_amd64.run --silent \
-  && chmod +x /tmp/ros-humble-bosdyn_msgs_4.0.2-jammy_amd64.run \
-  && ((yes || true) | /tmp/ros-humble-bosdyn_msgs_4.0.2-jammy_amd64.run) \
-  && rm /tmp/ros-humble-bosdyn_msgs_4.0.2-jammy_amd64.run
+RUN curl -sL https://github.com/bdaiinstitute/bosdyn_msgs/releases/download/4.1.0/ros-humble-bosdyn_msgs_4.1.0-jammy_amd64.run --output /tmp/ros-humble-bosdyn_msgs_4.1.0-jammy_amd64.run --silent \
```

Comment:
```suggestion
RUN curl -sL https://github.com/bdaiinstitute/bosdyn_msgs/releases/download/${SPOT_SDK_VERSION}/ros-humble-bosdyn_msgs_${SPOT_SDK_VERISON}-jammy_amd64.run --output /tmp/ros-humble-bosdyn_msgs_${SPOT_SDK_VERSION}-jammy_amd64.run --silent \
```

Reply:
Done in https://github.com/bdaiinstitute/spot_ros2/pull/495/commits/88f0b6d9cd1abeffe4d8d6f5d87c1f926ed25684.